js/components/feedback-box.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+
+import pureRender from 'pure-render-decorator'
+
+const UPDATE_INTERVAL = 1000
+
+@pureRender
+export default class FeedbackBox extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: props.initialFeedback || ""
+    }
+    this.updateText = this.updateText.bind(this)
+  }
+
+  clearUpdateTimer() {
+    if(this.updateTimer) {
+      clearTimeout(this.updateTimer)
```

Comment:
It seems you never clean `this.updateTimer` value. But it also seems that browser don't care if the value really reflects some existing timeout ID.